### DISCUSSION TITLE
what is recommended/default adminstyle for e107 admin area

### DISCUSSION THREAD

**Discussion by Jimmi08** [e107inc/e107#4935] [Q&A Core] [answered]
What is default adminstyle for admin bootstrap3 theme?

During installation:
`<core name="adminstyle">flexpanel</core>`

But in auth.php callback for setting it back to boostrap3 is:

`$core->update('adminstyle','infopanel');`

if it is not set
`if (!isset($pref['adminstyle'])) $pref['adminstyle'] = 'infopanel';`

Thanks

**Comment by Moc** [comment, marked as answer]
Infopanel is the default. The only admin theme currently allowed is bootstrap3.
